perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -360,13 +362,34 @@ def _ProcessMultiStreamResults(raw_result, operation, sizes,
         'Difference between first and last stream start/end times was %s and '
         '%s, which is more than %s of the benchmark time %s.',
         start_gap, stop_gap, MULTISTREAM_STREAM_GAP_THRESHOLD,
-        any_streams_active.duration)
+        (last_stop_time - first_start_time))
     metadata['stream_gap_above_threshold'] = True
 
-  records_in_interval = records[
-      analysis.FullyInInterval(records['start_time'],
-                               records['latency'],
-                               all_streams_active)]
+  # Find the indexes in each stream where all streams are active,
+  # following Python's [inclusive, exclusive) index convention.
+  active_start_indexes = []
+  for start_time in start_times:
```

Comment:
@yuyantingzero I looked at the docs, but I'm not sure it applies in this case. It looks like `where` gives you all of the indices where some condition is true, but I only want the first index. I could use `where` and then only take the first index, but that seems like extra work.